src/views/intro/intro.vue

```diff
@@ -45,22 +45,27 @@ import EnterEmail from '@/components/onboarding/email/enter-email-component.vue'
     IntroWin,
     EnterEmail,
   },
+  data() {
+    return {
+      buttonText: 'Next',
+    };
+  },
 })
 export default class Intro extends Vue {
   public introStep = 1;
   public totalSteps = 5;
 
   get label() {
-    return (this.introStep === 1 || this.introStep === this.totalSteps)
+    return this.introStep === 1 || this.introStep === this.totalSteps
       ? 'Get Started'
       : 'Next';
   }
 
   public nextStep() {
-    this.$set(this, 'buttonText', 'Next');
+    this.$set(this.$data, 'buttonText', 'Next');
```

Comment:
This fixed the runtime error about adding data variables that weren't set when the component was created.